components/MdInput.tsx

```diff
@@ -230,6 +232,7 @@ export const MdInput: React.FC<MdInputProps> = ({
         placeholder={placeHolder}
         style={height ? { height: height + 'px' } : undefined}
         data-testid="textbox"
+        onFocus={e => onFocus(e)}
```

Comment:
onFocus callback function gets called right? So can't you just do `onFocus={onFocus}` ?

Reply:
Yeah, I could've done that.